components/html-block/demo/html-block.html

```diff
@@ -46,9 +47,6 @@
 			});
 
 		</script>
-		<script type="module">
-			import '../../../tools/mathjax-test-context.js';
```

Comment:
This was happening too late in the process, so context was being added _after_ first render. The old attribute controller didn't care about that mis-ordering, but the context provider absolutely does.